setup/install/providers/aws/aws-eks.md

```diff
@@ -0,0 +1,204 @@
+---
+layout: single
+title:  "Amazon EKS"
+sidebar:
+  nav: setup
+---
+
+
+
+{% include toc %}
+
+
+
+
+
+> Before you proceed with installing Spinnaker on EKS, we strongly recommend that you familiarize yourself with [Amazon EKS concepts](https://docs.aws.amazon.com/eks/latest/userguide/getting-started.html)
+
+
+
+Below are the instructions to be followed if you want to configure [Kubernetes V2 (manifest based) Clouddriver](/setup/install/providers/kubernetes-v2) 
+to run Spinnaker on [Amazon EKS](https://aws.amazon.com/eks/).
```

Comment:
To be consistent w/ the other provider pages, these should explain how to configure Spinnaker to deploy to this platform, not necessarily onto it.

Reply:
This wasn't addressed